src/Birkhoff_polytope.jl

```diff
@@ -0,0 +1,453 @@
+"""
+	BirkhoffBLMO
+
+A bounded LMO for the Birkhoff polytope. This oracle computes an extreme point subject to  
+node-specific bounds on the integer variables.
+"""
+struct BirkhoffBLMO <: Boscia.BoundedLinearMinimizationOracle
+    append_by_column::Bool
+    dim::Int
+    lower_bounds::Vector{Float64}
+    upper_bounds::Vector{Float64}
+    int_vars::Vector{Int}
+    fixed_to_one_rows::Vector{Int}
+    fixed_to_one_cols::Vector{Int}
+    index_map_rows::Vector{Int}
+    index_map_cols::Vector{Int}
+    atol::Float64
+    rtol::Float64
+end
+
+BirkhoffBLMO(dim, lower_bounds, upper_bounds, int_vars; append_by_column = true) =
+    BirkhoffBLMO(
+        append_by_column,
+        dim,
+        lower_bounds,
+        upper_bounds,
+        int_vars,
+        Int[],
+        Int[],
+        collect(1:dim),
+        collect(1:dim),
+        1e-6,
+        1e-3,
```

Comment:
This should be default values but ultimately up to the user.

BirkhoffBLMO(...; rtol = 1e-3, atol = 1e-6)